transport/x/grpc/config.go

```diff
@@ -66,7 +66,11 @@ func TransportSpec(opts ...Option) yarpcconfig.TransportSpec {
 // All parameters of TransportConfig are optional. This section
 // may be omitted in the transports section.
 type TransportConfig struct {
-	Backoff yarpcconfig.Backoff `config:"backoff"`
+	ServerMaxRecvMsgSize int                 `config:"serverMaxRecvMsgSize"`
+	ServerMaxSendMsgSize int                 `config:"serverMaxSendMsgSize"`
+	ClientMaxRecvMsgSize int                 `config:"clientMaxRecvMsgSize"`
+	ClientMaxSendMsgSize int                 `config:"clientMaxSendMsgSize"`
```

Comment:
might be good for the units (bytes?) to be in the variable name.

Reply:
In the annotation I assume? @abhinav @breerly thoughts? In terms of the variables, this is matching grpc-go.

Reply:
@kriskowal I personally prefer this, we can talk about it this week before moving grpc out of x/ though